cfgov/paying_for_college/search_indexes.py

```diff
@@ -0,0 +1,28 @@
+from haystack import indexes
+
+from paying_for_college.models import School
+
+
+class SchoolIndex(indexes.SearchIndex, indexes.Indexable):
+    text = indexes.CharField(document=True, model_attr='primary_alias')
+    school_id = indexes.IntegerField(model_attr='school_id')
+    nicknames = indexes.CharField()
```

Comment:
I'm not super familiar with Haystack index definition, but I notice some minor differences here [with college-costs](https://github.com/cfpb/college-costs/blob/54004216e4dd0fc2c460858b1e2689eb29fd9676/paying_for_college/search_indexes.py#L9):

```diff
9c9
<     nicknames = indexes.CharField()
---
>     nicknames = indexes.CharField(model_attr='nicknames')
20,22d19
<     def prepare_nicknames(self, obj):
<         return obj.nicknames
< 
```

Do these do the same thing?

Reply:
I'm so glad you raised this. I thought this was a hidden error, because 'nicknames' is a model property, not a field, and I thought Haystack's model_attr option required the value to be a db field, the way a Django query does. In fact, model_attr [handles callables](https://django-haystack.readthedocs.io/en/v2.4.1/searchfield_api.html?highlight=model_attr#SearchField.model_attr), so this fix is not needed!